pkg/sriov/sriov.go

```diff
@@ -382,5 +407,10 @@ func (s *sriovManager) ResetVFConfig(conf *sriovtypes.NetConf) error {
 		return fmt.Errorf("failed to disable rate limiting for vf %d %v", conf.VFID, err)
 	}
 
+	// Reset link state to `auto`
```

Comment:
Do we need `if` condition here to make sure `conf.LinkState` is not empty?

Reply:
Not sure. I assume that `auto` is always the right default state to leave a VF in. Unconditional application will also revert to `auto` even when e.g. `NetworkAttachmentDefinition` is updated after the port is initialized but before the port is deleted, which seems like the right thing to do.

It's easy to add a conditional here, let me know if you still believe it's the right thing to do, and I will update the PR.

Thanks!

Reply:
@zshi-redhat please check above

Reply:
@booxter, one thing I'm not sure is that if a device doesn't support setting link state(whether it's auto, enable or disable), will it fail or ignore the netlink setting?

Reply:
@zshi-redhat oh that's a good point! maybe we can ignore the error there. Or introduce a conditional as you originally proposed and make it less magick-y. Let me respin the PR.

Reply:
@zshi-redhat let me know if the latest version satisfies your request.